src/components/shared/Description/index.jsx

```diff
@@ -1,5 +1,9 @@
 import styles from "./index.module.scss";
 
-const Description = ({ text }) => <p className={styles.description}>{text}</p>;
+const Description = ({ text, upperCase }) => (
+  <p className={upperCase ? styles.upperCaseDescription : styles.description}>
```

Comment:
You can use a helper library like [classnames](https://www.npmjs.com/package/classnames) for doing conditional styling, you'll find it that it increases readability